src/main/java/Echo.java

```diff
@@ -0,0 +1,18 @@
+import java.util.Scanner;
+
+public class Echo {
+    public static void echo(Scanner scanner)
+    {
```

Comment:
The opening brace for your methods should be standardised to be on the same line as the method name